### ISSUE TITLE
Can not access "structuring element / entry point" subcategories via keyboard in main navigation

### ISSUE THREAD

**Issue by timo-haas-gh** [shopware/platform#9861]
### Shopware Version

6.6.10.4

### Affected area / extension

Platform(Default)

### Actual behaviour

If you create a "structuring element / entry point" for your main navigation with subcategories you can hover it to access the subcategories but you are unable to access them via keyboard.

### Expected behaviour

You should be able to access the "structuring element / entry point" via keyboard.

### How to reproduce

1. create and activate a category with type "structuring element / entry point" for your main navigation
2. create and activate subcategories for the new category
3. Reload storefront -> you cant access the subcategories

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by ravenokko** [comment]
could not reproduce in `6.7.0.0` or `trunk`. Likely just missing a backport.

**Comment by dgrothaus-sw** [comment]
Thank you for reporting this issue to us and for the detailed reproduction steps. The issue is now fixed and will be available in the next 6.6 release.
